.github/workflows/integration.yml

```diff
@@ -9,9 +9,13 @@ on:
         description: Otomi installation profile
         default: full
         type: string
+      server_providers:
+        description: Server where Otomi will be installed
+        type: string
+        default: scaleway
       cluster_region:
```

Comment:
The cloud region is not necessary for dev purpose

Reply:
If it is something we don't want to support I can delete it.